background/services/analytics/index.ts

```diff
@@ -65,7 +65,7 @@ export default class AnalyticsService extends BaseService<Events> {
         hasDefaultOnBeenTurnedOn,
       })
 
-      await this.emitter.emit("enableDefaultOn", undefined)
+      //await this.emitter.emit("enableDefaultOn", undefined)
```

Comment:
comment? or maybe remove the line, if we want analytics we can create an issue to add it

Reply:
Removed